README.md

```diff
@@ -0,0 +1,34 @@
+# Brainkeeper
+
+[![Build Status](https://drone.dev.sart.solutions/api/badges/brainkeeper/frontend/status.svg)](https://drone.dev.sart.solutions/brainkeeper/frontend)
+[![codecov](https://codecov.io/gh/brainkeeper/frontend/branch/master/graph/badge.svg)](https://codecov.io/gh/brainkeeper/frontend)
+[![Docker Pulls](https://img.shields.io/docker/pulls/hmbrainkeeper/frontend.svg)](https://hub.docker.com/r/hmbrainkeeper/frontend/)
+[![License: MIT](https://img.shields.io/badge/License-MIT-brightgreen.svg)](https://opensource.org/licenses/MIT)
+
+[**Visit the latest stable version**](https://app.brainkeeper.sart.solutions/)
+
+---
+
+### Introduction
+This Project was started by a [group of students](https://github.com/orgs/brainkeeper/people) from Munich University of Applied Sciences together with the Digital Transformation Lab and the local Alzheimer's Association
```

Comment:
This _project_ ... from _the_ Munich ...

Reply:
Maybe: This project was started ... Sciences during the Software Engineering I lecture taking place in Winter 2019/20. It  resulted from a cooperation with the local Alzheimer's Association and the HM brought together by the Digital Transformation Lab.

Reply:
> during the Software Engineering I lecture taking place in Winter 2019/20

I think this is to much information that no one reading this README would need.

But I like your suggestion to change “together” to “in cooperation with”, sounds way better.